modules/library/main/reports/pmd-report.txt

```diff
@@ -0,0 +1,226 @@
+src/main/java/org/geotools/data/store/ContentFeatureSource.java:19:	wildcards: No Wildcard Imports
```

Comment:
This file should not have been committed I believe.

Reply:
whoops